.eslintrc.js

```diff
@@ -1,23 +1,46 @@
 module.exports = {
   env: {
     browser: true,
-    es6: true
+    es6: true,
   },
   extends: ['eslint:recommended', 'airbnb', 'plugin:prettier/recommended'],
   globals: {
     Atomics: 'readonly',
-    SharedArrayBuffer: 'readonly'
+    SharedArrayBuffer: 'readonly',
   },
   parser: 'babel-eslint',
   parserOptions: {
     ecmaFeatures: {
-      jsx: true
+      jsx: true,
     },
     ecmaVersion: 2018,
-    sourceType: 'module'
+    sourceType: 'module',
   },
   plugins: ['react-native', 'prettier'],
   rules: {
+    'no-restricted-imports': [
+      'error',
+      {
+        paths: [
```

Comment:
If you wanted to be even cooler (like, fang should deng cool), you could restrict request.js outside of utility files....